.github/workflows/testsuite.yml

```diff
@@ -45,13 +45,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        perl-version: ["5.32", "5.30", "5.28", "5.26", "5.24", "5.22", "5.20", "5.18", "5.16", "5.14", "5.12", "5.10"]
+        perl-version:
```

Comment:
someone needs to come up with a macro for this so we don't have to update it every year on all the CI workflows. something like perl-version: all

Reply:
it'd also be nice of course to have perl-version: ["all", "dev"]